backend/src/llm/openai_client.py

```diff
@@ -0,0 +1,27 @@
+# src/llm/openai_client.py
+import openai
+from src.utils import Config
+import logging
+
+config = Config()
+logger = logging.getLogger(__name__)
+
+class OpenAIClient:
+    def __init__(self, api_key: str):
+        self.api_key = config.openai_key
+        openai.api_key = self.api_key
+
+    def chat(self, model, messages, temperature=0, max_tokens=150):
+        try:
+            response = openai.ChatCompletion.create(
+                model=model,
+                messages=messages,
+                temperature=0,
```

Comment:
Hmm, am I seeing this correctly, you are passing in to the function temp and max tokens, but then hardcoding them here?

Reply:
@philbaxter  - I have removed the hard coded values.

Reply:
Hmm, still not sure its quite right, what are the two values you now pass into the chat method used for? I was expecting the two parameters passed into chat() to then be passed on to the create method, but it seems you have now removed them.

Reply:
@philbaxter  - The current 2 values passed are model & messages. Earlier I was passing Temp and Tokens but those have default values in the chat method.
i.e.
def chat(self, model, messages, temperature=0, max_tokens=150):

Reply:
@philbaxter  - Please have a look now and see if it looks good to you.